ReactWindows/ReactNative/Views/Image/ReactImageData.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace ReactNative.Views.Image
+{
+    class ReactImageData : IDisposable
```

Comment:
>ReactImageData [](start = 10, length = 14)

ReactImageManager is already a monolith. It probably wouldn't hurt for this class to be a nested class in ReactImageManager.

Also if possible, it would be good if we keep this as a "properties-only" class, that way the side-effects of the ViewManager lifecycle events and prop hooks are much more explicit.